### PR TITLE
Add multi-source audio output demo; add select to filter demo; fix AppRTC params link

### PR DESCRIPTION
**Pull request by samdutton**
Didn't squash commits, but hopefully that makes it easier to see what's been done.

### CODE REVIEW

File: src/content/devices/multi/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
```
Comment (KaptenJansson) [inline src/content/devices/multi/index.html]: Update Copyright year.

File: src/content/devices/multi/css/main.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+audio {
```
Comment (KaptenJansson) [inline src/content/devices/multi/css/main.css]: Add copyright header.

File: src/content/devices/multi/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+       </audio>
+     </div>
+
+   </div>
+
+   <p>Press play to select a source. Only one output can be selected for all sources.</p>
+
+   <div>
+    <label>Select audio output: </label>
+    <select id="audioOutput"></select>
```
Comment (KaptenJansson) [inline src/content/devices/multi/index.html]: No camelCase in HTML, should be audio-output.

File: src/content/devices/multi/index.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+     <div>
+       <h2>getUserMedia():</h2>
+       <video class="gum" title="From getUserMedia()" autoplay controls></video>
+       <audio class="gum" title="From getUserMedia()" controls></audio>
+     </div>
+
+     <div>
+       <h2>Local media files:</h2>
+       <video id="localVideo" title="From local video file" controls loop poster="images/poster.jpg" preload="metadata">
```
Comment (KaptenJansson) [inline src/content/devices/multi/index.html]: No camelCase, should be local-video.

On 2nd thought, I thought we checked that with htmlhint?

File: src/content/devices/multi/css/main.css (hunk 2 above)
Comment (samdutton) [inline src/content/devices/multi/css/main.css]: Added.

File: src/content/devices/multi/index.html (hunk 1 above)
Comment (samdutton) [inline src/content/devices/multi/index.html]: Done.

File: src/content/devices/multi/index.html (hunk 4 above)
Comment (samdutton) [inline src/content/devices/multi/index.html]: I've removed id="localAudio" and id="localVideo", since they're redundant.

**Comment by KaptenJansson** [comment]
Maybe add a [note](https://github.com/webrtc/samples/blob/master/src/content/devices/input-output/index.html#L68) about the reverb issue as I did on the input-output demo?

File: src/content/devices/multi/index.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    <label>Select audio output: </label>
+    <select id="audioOutput"></select>
+  </div>
+
+  <p>This demo must be run over HTTPS on Chrome (or Chrome Canary) 45.0.2441.x or later.<p>
+  <p><strong>Enable experimental Web Platform features</strong> must be enabled from the chrome://flags page or run Chrome with the following flag:</p>
+  <p>&nbsp;&nbsp;&nbsp;&nbsp;<code>--enable-blink-features=EnumerateDevices,AudioOutputDevices</code></p>
+  <p>More information about running Chrome with flags is available from <a href="https://www.chromium.org/developers/how-tos/run-chromium-with-flags" title="The Chromium Projects: Run Chromium with flags">The Chromium Projects</a>.</p>
+
+  <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/devices/multi" title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
```
Comment (KaptenJansson) [inline src/content/devices/multi/index.html]: I noticed we have gh-pages as a source link on all our demo pages, should we link to the master branch instead? If so we should do that in a separate PR.

**Comment by KaptenJansson** [comment]
I actually tinkered with setting the volume on the video element to reduce said reverb issue but since everyone is using different equipment and have different audio levels it's most likely moot.

LGTM apart from nits.

File: src/content/devices/multi/index.html (hunk 5 above)
Comment (samdutton) [inline src/content/devices/multi/index.html]: Yes — just noticed the same, not sure where this came from... I'll do a PR.

**Comment by samdutton** [comment]
> Maybe add a note about the reverb issue as I did on the input-output demo?

Might leave this out — there's already too much text :). Reverb is the mating call of the WebRTC client ;).

File: src/content/devices/multi/index.html (hunk 5 above)
Comment (samdutton) [inline src/content/devices/multi/index.html]: @KaptenJansson see #546. (Also includes a couple of commits that should have been pushed before I merged this PR :^\...)
